cluster/gce/configure-vm.sh

```diff
@@ -488,6 +488,7 @@ project-id = ${PROJECT_ID}
 EOF
     cat <<EOF >>/etc/salt/minion.d/grains.conf
   cloud_config: /etc/gce.conf
+  advertise_address: '${KUBERNETES_MASTER_NAME}'
```

Comment:
Right now, this is inside the "if (using alternate tokenSource)", which happens to be when we set KUBERNETES_MASTER_NAME to be an IP on GKE.

There's probably a better way to organize this, but this seemed expedient.